modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -32,77 +32,158 @@ on {rh-openstack-first}
 endif::[]
 to use an external load balancer in place of the default load balancer.
 
-You can also configure an {product-title} cluster to use an external load balancer that supports multiple subnets. If you use multiple subnets, you can explicitly list all the IP addresses in any networks that are used by your load balancer targets. This configuration can reduce maintenance overhead because you can create and destroy nodes within those networks without reconfiguring the load balancer targets.
+[IMPORTANT]
+====
+Configuring an external load balancer depends on your vendor's load balancer.
+
+The information and examples in this section are for guideline purposes only. Consult the vendor documentation for more specific information about the vendor's load balancer.
+====
+
+Red Hat supports the following services for an external load balancer:
+
+* OpenShift API
+* Ingress Controller
 
-If you deploy your ingress pods by using a machine set on a smaller network, such as a `/27` or `/28`, you can simplify your load balancer targets.
+You can choose to configure one or both of these services for an external load balancer, with configuring only the Ingress Controller service as a common configuration route.
```

Comment:
```suggestion
You can choose to configure one or both of these services for an external load balancer. Configuring only the Ingress Controller service is a common configuration option.
```
Suggest breaking up sentence and minor edits for readability

Reply:
Suggest using "option" throughout instead of "route" - might just be me, but when I read "route" in these docs I think of [these types of routes](https://docs.openshift.com/container-platform/4.13/networking/routes/route-configuration.html) and that may be confusing for some.